monai/handlers/clearml_handlers.py

```diff
@@ -0,0 +1,156 @@
+            tags:                       A list of tags (str) to the created Task.
+            reuse_last_task_id:         Force a new Task (experiment) with a previously used Task ID.
+            continue_last_task:         Continue the execution of a previously executed Task (experiment).
+            auto_connect_frameworks:    Automatically connect frameworks.
+        """
+
+        if Task.current_task():
+            self.clearml_task = Task.current_task()
+        else:
+            self.clearml_task = Task.init(
+                project_name=project_name,
+                task_name=task_name,
+                output_uri=output_uri,
+                tags=tags,
+                reuse_last_task_id=reuse_last_task_id,
+                continue_last_task=continue_last_task,
+                auto_connect_frameworks=auto_connect_frameworks,
+            )
+
+
+class ClearMLStatsHandler(ClearMLHandler, TensorBoardStatsHandler):
+    """
+
+    Class to write tensorboard stats by inheriting TensorBoardStatsHandler class.
+    Everything from Tensorboard is logged automatically to ClearML WebUI.
+
+    """
+
+    def __init__(
+        self,
+        project_name: str | None = "MONAI",
+        task_name: str | None = "Default Task",
+        output_uri: str | bool = True,
+        tags: Sequence[str] | None = None,
+        reuse_last_task_id: bool = True,
+        continue_last_task: bool = False,
+        auto_connect_frameworks: bool | Mapping[str, bool | str | list] = True,
+        *args,
+        **kwargs,
+    ):
+        """
+        Args:
+            project_name:               ClearML project name.
+            task_name:                  ClearML task name.
+            output_uri:                 The default location for output models and other artifacts.
+            tags:                       A list of tags (str) to the created Task.
+            reuse_last_task_id:         Force a new Task (experiment) with a previously used Task ID.
+            continue_last_task:         Continue the execution of a previously executed Task (experiment).
+            auto_connect_frameworks:    Automatically connect frameworks.
+        """
+
+        ClearMLHandler.__init__(
+            self,
+            project_name=project_name,
+            task_name=task_name,
+            output_uri=output_uri,
+            tags=tags,
+            reuse_last_task_id=reuse_last_task_id,
+            continue_last_task=continue_last_task,
+            auto_connect_frameworks=auto_connect_frameworks,
+        )
+        TensorBoardStatsHandler.__init__(self, *args, **kwargs)
+
+
+class ClearMLImageHandler(ClearMLHandler, TensorBoardImageHandler):
+    """
+
+    This class inherits all functionality from TensorBoardImageHandler class.
+    Everything from Tensorboard is logged automatically to ClearML WebUI.
+
+    """
+
+    def __init__(
+        self,
+        project_name: str | None = "MONAI",
+        task_name: str | None = "Default Task",
```

Comment:
Please refer to `MLFlowHandler` for the default names:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/handlers/mlflow_handler.py#L114

Thanks.

Reply:
Changed the "task_name" defaults to "monai_experiment" like MLFlow.